src/services/getBlockHash.ts

```diff
@@ -0,0 +1,65 @@
+import { Request, Response } from "express";
+import { Pool } from "pg";
+
+const query = `
+    SELECT encode(hash, 'hex') as hash, epoch_no, slot_no
+    FROM   block
+    WHERE  epoch_no <= $1 AND slot_no <= $2
```

Comment:
@AhmedIbrahim336 , note that slot `[1, 20]` is lower than slot `[2, 10]`, but your query will not select it.